smsdk/config/message_config.json

```diff
@@ -3,6 +3,7 @@
   "auth_header-api-secret_old": "X-SM-API-Key",
   "auth_header-api-key": "X-SM-API-Key-Id",
   "x_sm_db_schema":"x-sm-db-schema",
+  "x_sm_workspace_id":"X-Sm-Workspace-Id",
```

Comment:
I don't think the case matters, but can we change this to be uppercase just to be consistent with MA?

Reply:
I can but I changed the casing to be consistent with Fe-React